### PR TITLE
Added home-assistant.io to registry

### PR DESCRIPTION
**Pull request by KrayZee**
Home automation software written in Python3 (https://home-assistant.io/)
Supports many Smart Home hardware and software including Philips Hue, Google Chromecast, Plex, TP-Link Smart Plug, Vera, and many others.

### CODE REVIEW

**Comment by schakrava** [comment]
Hi @KrayZee . Thanks for introducing homeassistant. I did not know it existed. It looks pretty cool and I like that it's written in Python. Could you address my review comments?

**Comment by KrayZee** [comment]
Hi @schakrava 
I'd like to, but it looks I can't find any review comments.

**Comment by phillxnet** [comment]
@KrayZee Hello, just jumping in on this one as I also can't see any comments but pretty sure I saw some recently from @schakrava  relating to the redundancy of mapping /etc/localtime:
A similar comment re /etc/localtime is available as item 2 (13th July) here:
https://github.com/rockstor/rockon-registry/pull/48
and the other comment I seem to remember being about the --net=host option which we really don't want to use if we can at all help it. It was found that without this the plex rock-on just failed to work but many others are running without it, best if we can avoid it really.

I've found a similar @schakrava comment re --net=host here:
https://github.com/rockstor/rockon-registry/pull/45

Hope that helps.

**Comment by schakrava** [comment]
ok, my comments are visible now. I forgot to press a button, sorry.

File: home-assistant.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    },
+    "website": "https://home-assistant.io/",
+    "version": "1.0",
+    "containers": {
+      "home-assistant": {
+        "image": "homeassistant/home-assistant",
+        "launch_order": 1,
+        "opts": [
+          [
+            "--net=host",
```
Comment (KrayZee) [inline home-assistant.json]: according to documentation for home-assistant:
"Home Assistant must be on the same network as the devices for uPnP discovery to work. If running Home Assistant in a Docker container use switch --net=host to put it on the host’s network."
So removing this option will break discovery for devices such as Google Chromecast, Philips Hue, Plex Mediaserver, etc. I tried it without --net=host, rock-on starts, but discovery doesn't work indeed. So if it is possible i think it's better to keep this option

**Comment by KrayZee** [comment]
redundant mapping /etc/localtime is removed. Also --net=host seems to be necessary for application to work due to UPnP network discovery

File: home-assistant.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            "",
+            "-v",
+            "/etc/localtime:/etc/localtime:ro"
```
Comment (KrayZee) [inline home-assistant.json]: done

**Comment by schakrava** [comment]
Thank you @KrayZee !

**Pull request merged**